doi-requests/src/main/java/no/unit/nva/doirequest/list/ListDoiRequestsHandler.java

```diff
@@ -83,7 +84,7 @@ private static MessageService defaultMessageService(AmazonDynamoDB client, Clock
 
     @JacocoGenerated
     private static DoiRequestService defaultDoiRequestService(AmazonDynamoDB client, Clock clock) {
-        return new DoiRequestService(client, clock);
+        return new DoiRequestService(client, EXTERNAL_SERVICES_HTTP_CLIENT, clock);
```

Comment:
EXTERNAL_SERVICES_HTTP_CLIENT is the default HttpClient used for communicating with the Person proxy. For now it is used only the Person proxy but it could be used for other services as well.